core/domain/activity_jobs_one_off.py

```diff
@@ -704,7 +704,7 @@ def map(snapshot_model):
             yield ('Found commit log model-%s' % class_name, 1)
             return
 
-        if parent_model is None:
+        if parent_model is None or parent_model.deleted:
```

Comment:
Make a separate if and yield for the deleted case.

Reply:
Done.